debian/control

```diff
@@ -23,6 +23,7 @@ Homepage: https://github.com/linux-rdma/rdma-core
 Package: rdma-core
 Architecture: linux-any
 Depends: dmidecode, ${misc:Depends}, ${shlibs:Depends}
+Recommends: dmidecode, ethtool, net-tools
```

Comment:
You forgot to remove dmidecode from Depends. Please put the move of dmidecode in a separate commit.